feature-group-definitions/css-scoping.yml

```diff
@@ -0,0 +1 @@
+spec: https://drafts.csswg.org/css-cascade-6/#scope-atrule
```

Comment:
Hmm, `scoped-rules.yml`? `scoped-styles.yml`? Even `scopes.yml`, alone?

Reply:
The syntax is `@scope` so something with the word "scope" is also an option. Although that doesn't seem like the best choice in the case of `@layer` so I think consistency won't be possible everywhere.

Reply:
`scope` is good too! The main thing I want to steer away from is the `css` prefix. I'd rather we choose names that are familiar and match, as best we can, the names people actually use for those things.

(I also expect to add `shortName`, `longName`, `description`, and such so we have access to other words for presentation and search.)